app/models/api/mobile_tags_params.rb

```diff
@@ -8,7 +8,7 @@ module Types
 
   Schema =
     Dry::Validation.Schema do
-      required(:input).filled(:str?)
+      required(:input)
```

Comment:
is it still required though? what's the value when it's not filled with a string?

Reply:
It means you can pass `"input":""`, but the keyword input has to be in the params. It also means that in the code later I can do `form[:input]` without  checking if it exists.